### PR TITLE
Fix issues found by ansible-test and linters - enable all tests on all repos - remove suppressions

### PR DESCRIPTION
**Pull request by nhosoi**
- Disabling yamllint line-length check just on a violating task and eliminating "line-length: disable" in .yamllint.yml
- Cleaning up ansible-lint errors.
- Cleaning up ansible-test errors.

### CODE REVIEW

File: library/nbde_server_tang.py [hunk 1]
```diff
@@ -265,7 +271,7 @@ def update_cache(module, keydir, cachedir, update):
 
     if not module.check_mode:
         args = [update, keydir, cachedir]
-        ret, _, _ = module.run_command(args)
+        ret, _ret0, _ret1 = module.run_command(args)
```
Comment (richm) [inline library/nbde_server_tang.py:274]: use `_unused1, _unused2`

File: .sanity-ansible-ignore-2.10.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
-plugins/modules/nbde_server_tang.py pylint:blacklisted-name
-plugins/modules/nbde_server_tang.py pylint:ansible-format-automatic-specification
-plugins/modules/nbde_server_tang.py validate-modules!skip
+plugins/modules/nbde_server_tang.py validate-modules:doc-choices-do-not-match-spec
+plugins/modules/nbde_server_tang.py validate-modules:missing-gplv3-license
+plugins/modules/nbde_server_tang.py validate-modules:parameter-type-not-in-doc
+plugins/modules/nbde_server_tang.py validate-modules:undocumented-parameter
```
Comment (richm) [inline .sanity-ansible-ignore-2.10.txt:4]: `doc-choices-do-not-match-spec`, `parameter-type-not-in-doc`, `undocumented-parameter` are usually straightforward to fix - were you planning on a separate commit for those?

Comment (nhosoi) [reply .sanity-ansible-ignore-2.10.txt:4]: > doc-choices-do-not-match-spec, parameter-type-not-in-doc, undocumented-parameter are usually straightforward to fix - were you planning on a separate commit for those?

I was worried the fixes might be "invasive", but if they are straightforward, I'm going to look into them some more. Thanks.

Comment (richm) [reply .sanity-ansible-ignore-2.10.txt:4]: I think they are just doc fixes - make the docs embedded in the python file accurately describe the parameters in the python code.

**Pull request merged**